docs/querying/sql-window-functions.md

```diff
@@ -0,0 +1,164 @@
+| 2016-06-27T00:00:00.000Z	| #kk.wikipedia	| 7	| 3 |
+| 2016-06-27T00:00:00.000Z	| #kk.wikipedia	| 56 | 4 |
+| 2016-06-27T00:00:00.000Z	| #kk.wikipedia	| 56 | 4 |
+| 2016-06-27T00:00:00.000Z	| #kk.wikipedia	| 63 | 6 |
+| 2016-06-27T00:00:00.000Z	| #kk.wikipedia	| 91 | 7 |  
+| 2016-06-27T00:00:00.000Z	| #kk.wikipedia	| 2440 | 8 |
+| 2016-06-27T00:00:00.000Z	| #kk.wikipedia	| 2703 | 9 |
+| 2016-06-27T00:00:00.000Z	| #kk.wikipedia	| 6900 |10 |
+| 2016-06-27T00:00:00.000Z	| #lt.wikipedia	| 1	| 1 |
+| 2016-06-27T00:00:00.000Z	| #lt.wikipedia	| 2	| 2 |
+| 2016-06-27T00:00:00.000Z	| #lt.wikipedia	| 13 | 3 |
+| 2016-06-27T00:00:00.000Z	| #lt.wikipedia	| 28 | 4 |
+| 2016-06-27T00:00:00.000Z	| #lt.wikipedia	| 53 | 5 |
+| 2016-06-27T00:00:00.000Z	| #lt.wikipedia	| 56 | 6 |
+| 2016-06-27T00:00:00.000Z	| #lt.wikipedia	| 59 | 7 |
+| 2016-06-27T00:00:00.000Z	| #lt.wikipedia	| 391 | 8 |
+| 2016-06-27T00:00:00.000Z	| #lt.wikipedia	| 894	| 9 |
+| 2016-06-27T00:00:00.000Z	| #lt.wikipedia	| 4358 | 10 |
+
+</details>
+
+Window functions are similar to [aggregation funtctions](./aggregations.md).  
+
+You can use the OVER clause treat other Druid aggretation functions as window functions. For example, the sum of a value for rows within a window.
+
+When working with window functions, consider the following:
+- Window functions only work on GROUP BY queries.
+- Window functions support aliasing.
+
+## Define a window with the OVER clause
+
+The OVER clause defines the query windows for window functions as follows:
+- PARTITION BY indicates the dimension that defines the rows within the window
+- ORDER BY specifies the order of the rows within the windows.
+Sometimes windows are called partitions, but don't confuse them with the segment partitioning feature of Druid.
+
+
+The PARTITIONED BY clause defines the dimenson to use for the window. For example, the following OVER clause sets the window dimension to `channel` and orders the results by the absolute value of `delta` ascending:
+
+```sql
+...
+rank() OVER (PARTITION BY channel ORDER BY ABS(delta) ASC)
+...
+```
+
+## Window function reference
+
+|Function|Notes|Default|
+|--------|-----|-------|
+| `ROW_NUMBER()`| Returns the number of the row within the window| None |
+|`RANK()`| Returns the rank for a row within a window | None | 
+|`DENSE_RANK()`| Returns the ranks for a row within a window without gaps. For example, if two rows tie for rank of 1, the subsequent row is ranked 2. | None |
+|`PERCENT_RANK()`| Returns the rank of the row calculated as a percentage according to the formula: `(rank - 1) / (total partition rows - 1)` | None |
+|`CUME_DIST()`| Returns the cumulative distribution of the current row within the window calculated as `number of partition rows preceding or peers with current row` / `total partition rows` | None |
+|`NTILE(tiles)`| Divides the results as evenly as possible into the number of tiles, also called buckets, and returns a value from 1 to the value of `tiles`  |None |
```

Comment:
I'm not sure I understand what this function does even after rereading it a few times. Are the results divided per each window? If I call `NTILES(5)`, it returns a result from 1 to 5, but how do I know what will be returned?